src/globals.ts

```diff
@@ -0,0 +1,3 @@
+if (!global.Buffer) {
+    global.Buffer = require("buffer").Buffer;
```

Comment:
do we also need to polyfill stream here?

Reply:
Not needed (yet, at least). Will add when needed.